content/300-guides/050-database/900-turso.mdx

```diff
@@ -0,0 +1,163 @@
+---
+title: 'Using Prisma with Turso'
+metaTitle: 'Using Prisma with Turso'
+metaDescription: 'Guide to using Prisma with Turso'
+tocDepth: 3
+---
+
+<TopBlock>
+
+This guide discusses the concepts behind using Prisma and Turso, explains the commonalities and differences between Turso and other database providers, and leads you through the process for configuring your application to integrate with Turso
+
+</TopBlock>
+
+## What is Turso?
+
+[Turso](https://turso.tech/) is an edge-hosted, distributed database that's based on [libSQL](https://turso.tech/libsql), an open-source and open-contribution fork of [SQLite](https://sqlite.org/), enabling you to bring data closer to your application and minimize query latency.
```

Comment:
I'm not sure what makes this page boring but we have plenty of images in the section about embedded databases to make it less boring. 😛 

We can experiment with that in a separate PR — for all database guides we have 🙂

Reply:
Or we can just do that in a new page we add, instead of saying we will do it for all of them and then not do it for another 2 years as we did last time.